python/tvm/relay/op/strategy/arm_cpu.py

```diff
@@ -135,6 +136,14 @@ def conv2d_strategy_arm_cpu(attrs, inputs, out_type, target):
                     name="conv2d_direct_simd.micro_dev",
                 )
             elif kernel_layout == "HWIO":
+                if is_auto_scheduler_enabled():
```

Comment:
you forget depthwise convolution and winograd

Reply:
In x86, we have:
```python
                judge_winograd_auto_scheduler = (
                    "float" in data.dtype
                    and "float" in kernel.dtype
                    and kernel_h == 3
                    and kernel_w == 3
                    and stride_h == 1
                    and stride_w == 1
                    and dilation_h == 1
                    and dilation_w == 1
                    and 64 < co < 512
                    # The last condition of co is based on our profiling of resnet workloads
                    # on skylake avx512 machines. We found winograd is faster than direct
                    # only when co is within this range
                )
```
, do you have any suggestion on how to deal with the `co` in ARM? ....

As for the depthwise, I'm finding that we do not have Autoscheduler support for NHWC even in x86, only a NCHWc support.

Maybe these problems are better to be solved in another pr.

Reply:
For winograd, I have tested on ARM, the best value is ` 64 <= co < 512`.

For depthwise convolution,  we shouldn't enter into `NCHWc`.  And should be:
```python
        elif layout == "NHWC":
            assert kernel_layout == "HWOI"
            from tvm.auto_scheduler import is_auto_scheduler_enabled
            if is_auto_scheduler_enabled():
                strategy.add_implementation(
                    wrap_compute_conv2d(topi.nn.depthwise_conv2d_nhwc),
                    naive_schedule,
                    name="depthwise_conv2d_nhwc.arm_cpu",
                )
            else:
                strategy.add_implementation(
                    wrap_compute_conv2d(topi.arm_cpu.compute_depthwise_conv2d_nhwc),
                    wrap_topi_schedule(topi.arm_cpu.schedule_depthwise_conv2d_nhwc),
                    name="depthwise_conv2d_nhwc.arm_cpu",
                )
```

Reply:
Updated.

Reply:
Where is depthwise convolution of Auto Scheduler now?

Reply:
also add dense, batch_matmul?

Reply:
@jcf94  also add dense, batch_matmul?